resolv.cabal

```diff
@@ -78,7 +78,7 @@ library
                      Network.DNS.FFI
                      Compat
 
-  build-depends:     base              >= 4.5 && <4.11
+  build-depends:     base              >= 4.5 && <4.13
```

Comment:
I'm hesitant about relaxing the bound here as I don't want to give the false impression that GHC HEAD was supported already

Reply:
Ah OK, you're right. Lets only support GHC 8.4.3 for now.

Would it be OK to bump the version once it can actually build with GHC HEAD ? That requires relaxing the corresponding version bound on some of the dependencies (async, hashable, perhaps more). I can submit those PR's upstream (I already verified that hashable builds with GHC HEAD).